src/bundle-manifest.ts

```diff
@@ -64,32 +64,20 @@ export class BundleManifest {
   // Map of bundle url to bundle.
   bundles: Map<UrlString, Bundle>;
```

Comment:
If external writes to this map are allowed then it seems like we should document the contract a bit more.

When is it acceptable to mutate these bundles? May a file be in multiple bundles?